public/app/features/migrate-to-cloud/cloud/EmptyState/InfoPane.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+
+import { InfoItem } from '../../shared/InfoItem';
+import { MigrationTokenPane } from '../MigrationTokenPane/MigrationTokenPane';
+
+export const InfoPane = () => {
+  return (
+    <Box alignItems="flex-start" display="flex" padding={2} gap={2} direction="column" backgroundColor="secondary">
```

Comment:
We should:

 - remove the background colour from this element - it's no longer needed as the parent sets the background
 - remove the padding - it's not longer required to pad out the background colour any more, and consumers should be spacing elements apart.

I believe MigrationTokenPage should also do this.